Use `$(MAKE)` instead of `make` in mfile, the Perl script
WizardAux and the shell script TestScript.
--- ntl.orig/ntl/src/mfile	2014-10-11 07:33:02.904149666 +1300
+++ ntl/ntl/src/mfile	2014-10-09 22:14:46.730922903 +1300
@@ -325,11 +325,11 @@
 # again.
 
 all:
-	make setup1
-	make setup2
-	make setup3
-	make setup4
-	make ntl.a
+	$(MAKE) setup1
+	$(MAKE) setup2
+	$(MAKE) setup3
+	$(MAKE) setup4
+	$(MAKE) ntl.a
 	touch all
 
 
@@ -338,7 +338,7 @@
 setup1:
 	$(COMPILE) MakeDescAux.c
 	$(LINK) -o MakeDesc MakeDesc.c MakeDescAux.o $(LDLIBS)
-	./MakeDesc
+	./MakeDesc # generates mach_desc.h
 	mv mach_desc.h ../include/NTL/mach_desc.h

 
@@ -358,7 +358,7 @@
 # setup4 runs the wizard
 
 setup4:
-	sh Wizard $(WIZARD)
+	+sh Wizard $(WIZARD) # shell script invoking a Perl script invoking \$MAKE
 
 
 ntl.a:	$(OBJ) 
@@ -397,10 +397,10 @@
 
 check:
 	sh RemoveProg $(PROGS)
-	make QuickTest
+	$(MAKE) QuickTest
 	./QuickTest
 	sh RemoveProg QuickTest
-	sh TestScript
+	+sh TestScript # shell script invoking \$MAKE
 
 #################################################################
 #
--- ntl.orig/ntl/src/WizardAux	2014-08-27 05:51:43.000000000 +1200
+++ ntl/ntl/src/WizardAux	2014-10-09 22:14:46.730922903 +1300
@@ -47,9 +47,9 @@
    my $val;
    my $res;
 
-   system("make wntl.a") and return "999999999999999";
+   system("\$MAKE wntl.a") and return "999999999999999";
    RemoveProg($name);
-   system("make $name") and return "999999999999999";
+   system("\$MAKE $name") and return "999999999999999";
 
    print "\n*** running $name...";
 
@@ -73,7 +73,7 @@
 
 
 
-system("make InitSettings");
+system("\$MAKE InitSettings");
 
 @lines = `./InitSettings`;
 
@@ -254,7 +254,7 @@
 
 print "\n\n*** the wizard is done!!\n\n"; 
 
-system("make DispSettings");
+system("\$MAKE DispSettings");
 system("./DispSettings");
 
 
